### PR TITLE
fix for error when trying to run locally

### PR DESCRIPTION
**Pull request by bonham000**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -204,7 +204,20 @@ gulp
...
-congratulations – you're all set. Otherwise, let us know by asking in the [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) on Gitter. There also might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem.
+congratulations – you're all set. Otherwise, let us know by asking in the [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) on Gitter. There also might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem. If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, you may try the following:
+
+```bash
+# Remove all installed node modules
+rm -rf node_modules
+
+# Reinstall npm packages
+npm install
+# Re-run bower install
```
Comment (raisedadead) [inline CONTRIBUTING.md]: Add a new line before this line.

Comment (raisedadead) [inline CONTRIBUTING.md]: Also not concerning your changes, but advisable to add;

```markdown
# Seed the database (optional)
node seed
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Thanks for the changes, some minor comments and we are good to go.

You have edited from the GitHub Interface. Please close and reopen with valid branch naming. And filling up the checklist.

File: CONTRIBUTING.md (hunk 1 above)
Comment (QuincyLarson) [inline CONTRIBUTING.md]: I agree with @raisedadead on this. Once this is fixed I can merge.

**Comment by Bouncey** [comment]
Closed in favour of #11879

**Pull request closed without merging**